### ISSUE TITLE
Partial modal

### ISSUE THREAD

**Issue by Solido** [woltapp/wolt_modal_sheet#136]
## What feature would you like to see?

Wolt modal is doing correctly the modal display yet I'm facing a very specific case
where I need it to be partial.

For simplification, imagine a column with 2 entry, one Flexible and one Expanded.
Would it be possible to configure the Wolt modal by telling it its modal space is the Expanded area only?

Top package!

**Comment by ulusoyca** [comment]
Yes, it is possible. Please check this issue: https://github.com/woltapp/wolt_modal_sheet/issues/124

**Comment by Solido** [comment]
Thanks! I read that 124 concern the layout of the modal itself.
I may need to clarify that I want to keep the modal window intact but having the only background layer partial. Currently it use the full screen, probably rely on MediaQuery while I want to use only part of the screen.

**Comment by ulusoyca** [comment]
Persistent modal sheet is planned to be supported in later updates. Probably Q3
